stackle_api/server.js

```diff
@@ -42,4 +42,18 @@ mongoose.connect(database.url,function(err){
     }  
 });     // connect to mongoDB database on modulus.io
 
+// catch 404 and forward to error handler
+app.use(function(req, res, next) {
+    var err = new Error("Not Found");
+    err.status = 404;
+    next(err);
+});
+  
+// error handler
+app.use(function(err, req, res, next) {
+    // render the error page
+    res.status(err.status || 500);
+    process.env.NODE_ENV !== "DEVELOPMENT" ? res.sendFile(__dirname + "/public/404.html") : res.send(`<h1>Error Code: ${err.status || 500}</h1><br><p>${err.stack}</p>`);
```

Comment:
Can we show `404` when that page or resource is not found and show a `5xx` page for all the other errors?

Reply:
Yes, we can do that also.